doc/decisions/backend_plugin.md

```diff
@@ -2,35 +2,32 @@
 
 ## Problem
 
-- Backends store plugins in arrays which have a fixed number of slots for each plugin role. The number of plugins which can be assigned is limited,
-  making it easy to reach the limit if many plugins are in use.
-- As structs, backends are separate from the plugin interface and integrated into the core of Elektra. This makes it difficult to perform operations
-  such as nesting plugins, or to develop other implementations for backends.
+- Old backends (before Elektra 0.9.12) store plugins in arrays which have a fixed number of slots for each plugin role.
+  The number of plugins which can be assigned is limited, making it easy to reach the limit if many plugins are in use.
+- As structs, old backends are separate from the plugin interface and integrated into the core of Elektra.
+  This makes it difficult to perform operations such as nesting plugins, or to develop other implementations for backends.
 
 ## Constraints
 
-- It should be possible for all existing plugins to run normally
+- All existing plugins should continue working normally.
 
 ## Assumptions
 
 ## Considered Alternatives
 
 - Multiple storage plugins within a single backend
-- Plugin containing more plugin slots
+- Improve backend to contain more plugin slots
+- Making number of plugins per slot unlimited
 
 ## Decision
 
-- The current backend implementation will be redeveloped into a backend plugin. That way, the core of Elektra will only access backends through
-  the standard plugin interface.
-- The new backend plugin will support an unlimited number in any position where more than one plugin is sensible.
-  e.g., unlimited plugins in `poststorage`, but only a single one in `storage`
+- The current backend implementation will be redeveloped into a backend plugin. That way, the core of Elektra will only access backends through the standard plugin interface.
+- The new backend plugin will support an unlimited number of plugins in any position where more than one plugin is sensible, e.g., unlimited plugins in `poststorage`, but only a single one in `storage`
```

Comment:
Unclear if the "e.g." also refers to storage? If there are more exceptions than storage+resolver, it would be good to link there to the docu.

```suggestion
- The new backend plugin supports an unlimited number of plugins in any position where more than one plugin is sensible, e.g., unlimited plugins in `poststorage`, but only a single one in `storage` and `resolver`
```